aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -658,6 +658,11 @@ module aptos_framework::staking_contract {
         let (active, _, pending_active, _) = stake::get_stake(staking_contract.pool_address);
         let total_active_stake = active + pending_active;
         let accumulated_rewards = total_active_stake - staking_contract.principal;
+        spec {
+            // To avoid timeout, we assume staking_contract.commission_percentage is in range (0..100).
+            // This assumption is reasonable becasue a percentage value should be restricted in this range.
+            assume staking_contract.commission_percentage >= 0 && staking_contract.commission_percentage <= 100;
```

Comment:
I think we can prove it instead of assuming it because there is an assertion to ensure it.

Reply:
Oh, however, we tried to use invariant and require, but it caused a timeout, is there anyway to solve it without assume?